pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3394,6 +3394,37 @@ int tracepoint__sched__sched_switch(struct bpf_raw_tracepoint_args *ctx)
     return events_perf_submit(&data, SCHED_SWITCH, 0);
 }
 
+// based on /sys/kernel/tracing/events/syscalls/sys_enter_ptrace/format
+struct ptrace_args {
+    u64 __unused__[2]; // 16 bytes buffer
+    long request;
+    long pid;
+    unsigned long addr;
+    unsigned long data;
+};
+SEC("tracepoint/sys_enter_ptrace")
```

Comment:
For performance reasons, it is better to use raw_tracepoint and not tracepoint.
Changing to raw_tracepoint you will also be able to remove the ptrace_args defined here.